packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -71,6 +76,15 @@ const defaultProps = {
   children: null,
 };
 
+const contextTypes = {
```

Comment:
I am still curious, with adding `contextTypes`, do we still need to pass in the `locale` prop.